### DISCUSSION TITLE
'SCRAMBLE' STUTTER (RANDOM PLAYHEAD START)

### DISCUSSION THREAD

**Discussion by worker-gray** [SynthstromAudible/DelugeFirmware#2297] [Ideas]
This would be a new stutter type effect where the playhead would start at randomly chosen places on all effected tracks for as long as the effect was operating. Each activation would re-randomize playhead positioning, yeilding a new, unique stutter each time. Aside from random positioning, it would function the same-- it would still be rooted in specific intervals.

**Comment by monkeymanford** [comment]
it could be linked to the current step length of the grid so as to not have to set extra options, for instance if you're viewing in 16th steps, it would randomizes up to 16 steps around the playhead of each effected track.

**Reply by worker-gray** [reply]
EXACTLY! As long as it sounds different each time you engage the effect. I don't know how it would be accomplished with multiple-length patterns-- i suppose it would default to the number of steps in the longest pattern currently under the playhead and increase the iterations for the shorter patterns until they reach step parity- or near parity

**Reply by worker-gray** [reply]
Alternately it could default to 16 steps. Once the effect is engaged the listener/performer will lose track of time unless they're counting very carefully.

**Comment by monkeymanford** [comment]
Here is some more food for thought => you can choose between:
1 - 2 - 3 - ... - 16 - RND - RNDR.
The number represents how many steps are skipped for each "stutter", 
when set to "RND" it is random as you suggested,
when set to "RNDR" (stands for random-repeat) it will create a random pattern over 1 bar and then loop that pattern.
This way you could use the same logic and accomplish more predictable/musical patterns if desired.

This could be accessed by long-pressing or double tapping the "stutter/custom1" button.

Even more ideas:
- when getting into this menu, the columns of the grid become grid steps that you want repeated, so if you hold step 1, 5 and 7, it would loop over those three steps.

**Reply by worker-gray** [reply]
aww hell yeah! that's the stuff right there! great ideas! I hope somebody who can do this comes across this thread.
